src/backend/distributed/operations/node_protocol.c

```diff
@@ -623,10 +662,6 @@ GetTableCreationCommands(Oid relationId, bool includeSequenceDefaults)
 	{
 		tableDDLEventList = lappend(tableDDLEventList, tableOwnerDef);
 	}
-
-	/* revert back to original search_path */
-	PopOverrideSearchPath();
```

Comment:
is it ok that we don't override the search_path when calling this function? what did we need that for in GetTableCreationCommands?

Reply:
Since we're creating the table locally, omitting schemas that are in the search_path wouldn't lead to issues for the current PR, but maybe consider adjusting or adding a comment in case we use this function for a different purpose in the future.